src/main/java/com/megastore/controller/ProductController.java

```diff
@@ -29,12 +31,31 @@ public ProductController(ProductFacade productFacade) {
 
     @GetMapping("/hot")
     public ResponseEntity<List<ProductHotDto>> getHotProducts(@RequestParam(defaultValue = "4") int limit, @RequestParam(defaultValue = "1") long catId) {
-        return ResponseEntity.ok(new ArrayList<>(productFacade.findHotProducts(limit,catId)));
+        return ResponseEntity.ok(new ArrayList<>(productFacade.findHotProducts(limit, catId)));
+    }
 
+    @GetMapping("/list")
+    public ResponseEntity<Collection<ProductPLPDto>> getAllProducts(
+            @RequestParam(defaultValue = "9") int limit,
+            @RequestParam(defaultValue = "1") long subcategoryId,
+            @RequestParam(defaultValue = "UPDATED") Sorting sort,
+            @RequestParam(defaultValue = "0.00") Double priceFrom,
+            @RequestParam(defaultValue = "1000000.00") Double priceTo,
```

Comment:
you shouldn't have a default value for price. If there is no price, you need to return all items, even if the price is over 100000.